checkout-core/src/main/java/com/adyen/checkout/core/exception/PermissionException.kt

```diff
@@ -0,0 +1,14 @@
+/*
+ * Copyright (c) 2022 Adyen N.V.
+ *
+ * This file is open source and available under the MIT license. See the LICENSE file for more info.
+ *
+ * Created by atef on 16/11/2022.
+ */
+
+package com.adyen.checkout.core.exception
+
+class PermissionException constructor(
+    errorMessage: String,
+    val askedPermission: String
```

Comment:
Maybe `requiredPermission` or something could be clearer here? Also just a small doc on top of the class to explains what is this exception would be nice.

Reply:
What i meant by docs was docs for the merchants, to understand what this exception means, not internal docs for us.

Reply:
ohh sorry about that!